runit.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
will this work on our windows machines? if not, we will have to come up with a workaround for development